mediachain/indexer/mc_ingest.py

```diff
@@ -295,7 +296,7 @@ def cache_image(_id,
         
         url = image_cache_host + 'hh_' + size + '/' + _id[:3] + '/' + _id + '.jpg'
         
-        hsh = hashlib.md5(image_base64).hexdigest()
+        hsh = hashlib.md5(image_origin.encode('utf-8')).hexdigest()
```

Comment:
Ok, if we want to be sticklers about hashing stuff, we should not rely on the hash of the ur*L* (I have actually been bitten by this at hype machine, some sources reused urls for "track of the week" type links)

I guess it's OK in this case because both id AND url must match, but we really should be doing content hashing, not location (tho we could bloom filter based on location)